kore/src/Kore/Repl/Data.hs

```diff
@@ -168,7 +168,7 @@ helpText =
     \\n\
     \Available modifiers:\n\
     \<command> > file                    prints the output of 'command' to file\n\
-    \<command> | external script         pipes command to external script and prints the result in the repl\n\
+    \<command> | external script         pipes command to external script and prints the result in the repl (each command line argument should be between double quotes)\n\
```

Comment:
Looks like the line is too long, one below it too I think. Please check and split them on multiple lines if so